igvm/vm.py

```diff
@@ -1005,19 +1006,33 @@ def aws_get_instances_overview(
                 etag = None
 
             if file.exists() and etag_file.exists() and etag:
-                with open(AWS_INSTANCES_OVERVIEW_FILE_etag) as f:
+                with open(AWS_INSTANCES_OVERVIEW_FILE_etag, 'r+') as f:
```

Comment:
`AWS_INSTANCES_OVERVIEW_FILE_etag` should be `AWS_INSTANCES_OVERVIEW_FILE_ETAG`

Reply:
renamed